### ISSUE TITLE
building on orion 

### ISSUE THREAD

**Issue by JessicaMeixner-NOAA** [ufs-community/ufs-weather-model#405]
## Description

When building on orion with the commit e3983a03a62a0f77286495ce227a0c94d2b7ee2a I am getting an error that: 
```
CMake Error at /apps/cmake-3.17.3/share/cmake-3.17/Modules/FindPackageHandleStandardArgs.cmake:164 (message):
  Could NOT find Python (missing: Python_EXECUTABLE Interpreter) (Required is
  at least version "3")

      Reason given by package: 
          Interpreter: Wrong version for the interpreter "/bin/python"
```

### To Reproduce:
On orion with intel compilers: 

git clone https://github.com/ufs-community/ufs-weather-model 
cd ufs-weather-model 
git checkout e3983a03a62a0f77286495ce227a0c94d2b7ee2a
git submodule update --init --recursive 
module purge 
module use $path-to-ufs-weather-model/modulefiles/orion.intel/
module load fv3 
CMAKE_FLAGS="-DS2S=ON -DWW3=ON" CCPP_SUITES="FV3_GFS_v15p2_coupled,FV3_GFS_v16_coupled,FV3_GFS_v16" ./build.sh

## Additional context
This is basically taking out how I am trying to build ufs-weather-model for the coupled global-workflow.  It looks to me like perhaps I need to load a python 3 module?  This worked on hera yesterday, but it's down today so I cannot provide details for that right now. 

## Output

**output logs**
The output log from the workflow is here: /work/noaa/marine/jmeixner/p6workflow/global-workflow/sorc/logs/build_ufs_coupled.log

**Comment by junwang-noaa** [comment]
@BinLiu-NOAA @climbfuji @DusanJovic-NOAA Jessica created a ticket.

**Comment by junwang-noaa** [comment]
@JessicaMeixner-NOAA Please add:
module load python/3.7.5
after module purge. We will have a fix for this soon.

**Comment by JessicaMeixner-NOAA** [comment]
Thanks @junwang-noaa

**Comment by DeniseWorthen** [comment]
Fixed w/ PR #401
